portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,185 @@
+             </h4>
+                
+        </div>    
+    </div>    
+  </div>
+  
+  <!-- Projects -->
+  <div class="w3-container" id="projects" style="margin-top:75px">
+    <h1 class="w3-xxxlarge w3-text-teal"><b>Projects</b></h1>
+    <hr style="width:50px;border:5px solid teal" class="w3-round">
+    <h4>  During my time so far at Carnegie Mellon University, I have had the wonderful opportunity to take courses covering a wide range of fascinating topics in the field of computer science.
+         Many of these courses have culminated with a final project, where I have been able to gain a deeper understanding of the course concepts by researching, designing, and implementing algorithms and analyses on real-world data and frameworks.     
+     </h4>
+
+  </div>
+
+  <!-- The Projects -->
+  <div class="w3-row-padding">
+    <div class="w3-col m4 w3-margin-bottom">
+      <div class="w3-light-grey">
+        <img src="images/spectral.png" alt="Spectral" style="width:100%">
+        <div class="w3-container">
+          <h4>Spectral Clustering: A Python Implementation </h4>
+          <p class="w3-opacity">Linear Algebra - Fall 2018 &nbsp <a href="https://github.com/PriyaVarra/Spectral-Clustering-A-Python-Implementation"> <img src="images/GitHub-Mark-32px.png" width="32" height="32"> </a>
+          </p>   
+          <p> Spectral clustering is an unsupervised learning algorithm that clusters data by performing dimensionality reduction before clustering the resulting lower-dimensional data. In this project, we used Python to reduce dimensionality by constructing a similarity matrix from our data and taking its top eigenvectors. We then implemented k-means clustering and ran it on the selected eigenvectors to produce clusters for the original data. </p>
+        </div>
+      </div>
     </div>
-  </body>
+
+    <div class="w3-col m4 w3-margin-bottom">
+      <div class="w3-light-grey">
+        <img src="images/Figure2.png" alt="wbc" style="width:100%">
+        <div class="w3-container">
+          <h4> White Blood Cell Image Classification </h4>
+          <p class="w3-opacity">Computational Biology - Spring 2019 &nbsp <a href="https://github.com/PriyaVarra/Classifying-White-Blood-Cell-Images-Using-Deep-Learning"> <img src="images/GitHub-Mark-32px.png" width="32" height="32"> </a> </p>
+          <p> In this project, I attempted to create a computational model capable of classifying an image of a stained white blood cell into one of the four major types of white blood cells. Specifically, I used Python to preprocess a set of publicly available training images before using Tensorflow and Keras to create, train, and test three convolutional neural networks with the best model achieving an accuracy of 78%.   </p>
```

Comment:
I am not sure if this is just not being translated properly in my review page, but would it be possible to break this long line into multiple lines for easier reading?

Reply:
I completely agree. I have fixed this in the most recent version.